src/components/PageHeader/PageHeader.module.css

```diff
@@ -102,8 +103,9 @@
  */
 .page-header__description {
   @mixin reset;
+  @mixin eds-typography-preset-009;
```

Comment:
Let's not use tier 1 presets for page header descriptions. Let's also let `Text` do the heavy lifting here.